solidity/depool/README.md

```diff
@@ -394,9 +400,11 @@ all `--value` parameters must be defined in tons, like this: `--value 10.5`, whi
 
 `total <days>` - total period, for which the stake is made. 0 <`total`< 18 years.
 
-`withdrawal <days>` - withdrawal period (each time a withdrawal period ends, a portion of the stake is released to the beneficiary). Total period should be exactly divisible by withdrawal period.
+`withdrawal <days>` - withdrawal period (each time a withdrawal period ends, a portion of the stake is released to the beneficiary).
+
+> There are limitations for period settings: `withdrawalPeriodshould` be < `totalPeriod`, `totalPeriod` cannot exceed 18 years or be <=0, `totalPeriod` should be exactly divisible by withdrawalPeriod.
```

Comment:
typo `withdrawalPeriodshould`

Reply:
`withdrawalPeriod` should be <= `totalPeriod`

Reply:
fixed